src/Symfony/Bundle/DependencyInjection/ApiPlatformExtension.php

```diff
@@ -120,6 +120,9 @@ public function load(array $configs, ContainerBuilder $container): void
         $patchFormats = $this->getFormats($config['patch_formats']);
         $errorFormats = $this->getFormats($config['error_formats']);
         $docsFormats = $this->getFormats($config['docs_formats']);
+        $jsonSchemaFormats = $config['jsonschema_formats']
+            ? $this->getFormats($config['jsonschema_formats'])
+            : array_filter($formats, static fn ($format) => !\in_array($format, ['html', 'multipart', 'xml'], true), \ARRAY_FILTER_USE_KEY);
```

Comment:
mhh maybe that we could use a list of documentation-allowed formats instead? `jsonld, jsonapi, jsonhal` wdyt?

Reply:
OK, if it was my code I'd jump at the "documentation-allowed formats" … until I use it on *"that project"* where you need a custom format, and then wonder why it won't fit with my idea of what I want :innocent:  

I don't like this approach, I don't want to defend it, but we do know what defaults we already have that are not "distinct". 100% happy to apply a better solution though.